core/cmd/remote_client_test.go

```diff
@@ -99,18 +99,20 @@ func TestClient_IndexJobRuns(t *testing.T) {
 	j := cltest.NewJobWithWebInitiator()
 	assert.NoError(t, app.Store.CreateJob(&j))
 
-	jr0 := cltest.CreateJobRunViaWeb(t, app, j, `{"result":"100"}`)
-	jr1 := cltest.CreateJobRunViaWeb(t, app, j, `{"result":"105"}`)
-	jr2 := cltest.CreateJobRunViaWeb(t, app, j, `{"result":"110"}`)
+	jr0 := j.NewRun(j.Initiators[0])
```

Comment:
The new job runner is faster, which created a race condition here, the jobs would complete between them being created here and queried below.

Reply:
Here we create the jobs in a way which won't run them and change their state after the fact.